website/package.json

```diff
@@ -1,12 +1,17 @@
 {
     "name": "@goauthentik/website-docs",
+    "description": "Documentation for authentik",
     "version": "0.0.0",
     "private": true,
     "license": "MIT",
     "scripts": {
         "aws-cfn": "cd docs/install-config/install/aws && cdk synth > template.yaml && ./fix_template.py",
-        "build": "cp ../docker-compose.yml static/docker-compose.yml && cp ../schema.yml static/schema.yaml && docusaurus gen-api-docs all && cross-env NODE_OPTIONS='--max_old_space_size=65536' docusaurus build",
-        "build-bundled": "cp ../schema.yml static/schema.yaml && docusaurus gen-api-docs all && cross-env NODE_OPTIONS='--max_old_space_size=65536' docusaurus build",
+        "build": "run-s build:docker-compose build-bundled",
+        "build-bundled": "run-s build:schema build:api-docs build:docusaurus",
+        "build:api-docs": "docusaurus gen-api-docs all",
+        "build:docker-compose": "cp ../docker-compose.yml static/docker-compose.yml",
+        "build:docusaurus": "cross-env NODE_OPTIONS='--max_old_space_size=65536' docusaurus build",
+        "build:schema": "cp ../schema.yml static/schema.yml",
```

Comment:
@rissson Found the source of the mystery `.yaml` file 😊

I've tidied up the build sub-scripts to aid in readability.